aspnet/conceptual-overview/aspnet.rst

```diff
@@ -1,37 +1,37 @@
 ============================
 
-By `Daniel Roth`_
+By `Daniel Roth`_ and `Rick Anderson`_
 
 ASP.NET Core is a significant redesign of ASP.NET. This topic introduces the new concepts in ASP.NET Core and explains how they help you develop modern web apps.
 
 .. contents:: Sections:
   :local:
   :depth: 1
-  
+
 What is ASP.NET Core?
 ---------------------
 
-ASP.NET Core is a new open-source and cross-platform framework for building modern cloud-based Web applications using .NET. We built it from the ground up to provide an optimized development framework for apps that are either deployed to the cloud or run on-premises. It consists of modular components with minimal overhead, so you retain flexibility while constructing your solutions. You can develop and run your ASP.NET Core applications cross-platform on Windows, Mac and Linux. ASP.NET Core is fully open source on `GitHub <https://github.com/aspnet/home>`_.
+ASP.NET Core is a new open-source and cross-platform framework for building modern cloud-based Web apps using .NET. We built it from the ground up to provide an optimized development framework for apps that are deployed to the cloud or run on-premises. It consists of modular components with minimal overhead, so you retain flexibility while constructing your solutions. You can develop and run your ASP.NET Core apps cross-platform on Windows, Mac and Linux. ASP.NET Core is open source at `GitHub <https://github.com/aspnet/home>`_.
 
 Why build ASP.NET Core?
 -----------------------
 
-The first preview release of ASP.NET came out almost 15 years ago as part of the .NET Framework.  Since then millions of developers have used it to build and run great web applications, and over the years we have added and evolved many, many capabilities to it.
+The first preview release of ASP.NET came out almost 15 years ago as part of the .NET Framework.  Since then millions of developers have used it to build and run great web apps, and over the years we have added and evolved many capabilities to it.
 
-With ASP.NET Core we are making a number of architectural changes that make the core web framework much leaner and more modular. ASP.NET Core is no longer based on System.Web.dll, but is instead based on a set of granular and well factored NuGet packages allowing you to optimize your app to have just what you need. You can reduce the surface area of your application to improve security, reduce your servicing burden and also to improve performance in a true pay-for-what-you-use model.
+ASP.NET Core has a number of architectural changes that result in a much leaner and modular framework. ASP.NET Core is no longer based on *System.Web.dll*, but is instead based on a set of granular and well factored `NuGet <http://www.nuget.org/>`__ packages allowing you to optimize your app to include just the NuGet packages you need. You can reduce your apps surface area to improve security, reduce  servicing and improve performance in a pay-for-what-you-use model.
```

Comment:
@danroth27 can you come up with perf line? Increase RSP, less memory, faster startup ????
@Eilon is pretty conservative on any perf claims we assert.